app/src/main/java/com/example/mycalculator/mvp/CalculatorContract.java

```diff
@@ -0,0 +1,18 @@
+package com.example.mycalculator.mvp;
```

Comment:
you can create a package called contract inside mvp, and put this class inside

Reply:
Done. Thx!